mockingbird/src/commonMain/kotlin/com/careem/mockingbird/test/Capture.kt

```diff
@@ -47,18 +49,59 @@ public fun <T> capture(list: CapturedList<T>): CapturedMatcher<T> {
  */
 
 public class Slot<T> : Captureable {
+
+    private val genericSlot : GenereicSlot<T> = initializeSlot<T>()
+
+    public val captured: T?
+        get() {
+            return genericSlot.value
+        }
+
+    @Suppress("UNCHECKED_CAST")
```

Comment:
No casts here?

Reply:
yes I think the annotation is not needed here